manager-bundle/src/HttpKernel/ContaoKernel.php

```diff
@@ -171,6 +171,27 @@ public function setManagerConfig(ManagerConfig $managerConfig): void
         $this->managerConfig = $managerConfig;
     }
 
+    public static function getEnvVar(string $name, string $default = null, $deprecatedName = ''): ?string
```

Comment:
why does `$deprecatedName` not have a type?

Reply:
Fixed.